{{cookiecutter.project_slug}}/config/settings/test.py

```diff
@@ -29,5 +29,13 @@
 # ------------------------------------------------------------------------------
 TEMPLATES[0]["OPTIONS"]["debug"] = True  # type: ignore # noqa F405
 
+{%- if cookiecutter.frontend_pipeline == 'Webpack' %}
+# django-webpack-loader
+# ------------------------------------------------------------------------------
+WEBPACK_LOADER["DEFAULT"][  # noqa F405
+    "LOADER_CLASS"
+] = "webpack_loader.loader.FakeWebpackLoader"
```

Comment:
I was surprised to see that they had already had an loader for this, how could I have missed that!? Great job for fixing it upstream!

Reply:
I've just added it :)

Reply:
I like it